OutOfSchool/OutOfSchool.DataAccess/Models/ContactInfo/PhoneNumber.cs

```diff
@@ -3,16 +3,14 @@
 using OutOfSchool.Common;
 using OutOfSchool.Common.Validators;
 
-namespace OutOfSchool.Services.Models;
+namespace OutOfSchool.Services.Models.ContactInfo;
 
-public class PhoneNumber : IKeyedEntity<Guid>
+public class PhoneNumber
 {
-    public Guid Id { get; set; }
+    public string Type { get; set; } = null!;
```

Comment:
Please avoid usage of `null!` Here you can use `required` keyword
`public required string Type { get; set; }`

Reply:
removed `null!`, required is set via fluent api